provider/datagen/src/transform/cldr/displaynames/language_displaynames.rs

```diff
@@ -0,0 +1,96 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
```

Comment:
The module (file) name is repetitive. It should be `displaynames::language`.

Reply:
Done!